src/tcgwars/logic/impl/gen3/LegendMaker.groovy

```diff
@@ -1547,10 +1565,10 @@ public enum LegendMaker implements LogicCardInfo {
         pokeBody "Reactive Aroma", {
           text "As long as Roselia has any React Energy cards attached to it, remove 1 damage counter from each of your Pokémon (excluding Pokémon-ex) that has any React Energy cards attached to it between turns. You can't use more than 1 Reactive Aroma Poké-Body each turn."
           delayedA {
-            before BETWEEN_TURNS, {
-              if (self.cards.findAll{it.name.contains("React Energy")}) {
+            before BEGIN_TURN, {
+              if (self.cards.any{it.name == "React Energy"}) {
```

Comment:
I think this may need to be checked that it is only applying one of this ability each turn using a storedObject.

Reply:
Yup, good catch!